api/apps/geoprocessing/src/modules/scenarios/scenarios.service.ts

```diff
@@ -34,9 +34,10 @@ const includeSelections: IncludeSelections = {
   },
   features: {
     attributes: ', "featureList"',
-    select: 'array_to_string(feature_list, \',\') as "featureList"',
+    select: 'array_to_string(feature_list, \';\') as "featureList"',
```

Comment:
related to my note above, please consider if we should keep the existing `features` query as is and add the new "features+amounts" property as a new `include=featuresAndAmounts` kind of query param

Reply:
as discussed with FE, we will consider this if payload size is affected significantly when all we need in the app is feature lists (rather than features+amounts)